betelgeuse/parser.py

```diff
@@ -187,3 +187,37 @@ def parse_docstring(docstring=None):
             field_value = output
             fields_dict[field_name] = field_value
     return fields_dict
+
+
+def parse_markers(all_markers=None, config=None):
+    """Parse the markers from module, class and test level for a test.
+
+    This removes the mark prepended words and also pops out the
+    ignorable marker from the list received from the config object.
+
+    :returns string: Comma separated list of markers from all levels for a test
+    """
+    resolved_markers = []
+    ignore_list = getattr(config, 'MARKERS_IGNORE_LIST', None)
+
+    def _process_marker(marker):
+        # Fetching exact marker name
+        marker_name = marker.split('mark.')[-1]
+
+        # ignoring the marker if in ignore list
+        if ignore_list and any(
+                ignore_word in marker_name for ignore_word in ignore_list):
```

Comment:
Sorry, I only realized this now, but better later than never:
why do you do substring matching instead of full-string one?

```
In [1]: ignore_list = ['content_host']
   ...: marker_name = 'host'
   ...:
   ...: if ignore_list and any(ignore_word in marker_name for ignore_word in ignore_list):
   ...:     #return
   ...:     print('skipping')
   ...: #resolved_markers.append(marker_name)
   ...: print(f'appending {marker_name} to the list')
appending host to the list
```
^ is this the desired behavior? I am wondering what the motivation was here. Can't this backfire?


```suggestion
        if ignore_list and marker_name in ignore_list:
```